src/compiler/utilities.ts

```diff
@@ -2526,8 +2482,12 @@ namespace ts {
                     case SyntaxKind.CommaToken:
                         return 0;
 
+                    case SyntaxKind.AsteriskAsteriskToken:
+                        // This would be 15 if we used the below line instead, which changes emit for the test `emitExponentiationOperator4`.
+                        return 14;

```

Comment:
Its probably ok to bump this up to 15 since that reflects the current spec, though we should probably also bump the precedence of anything 15 or above up by one.